### PR TITLE
fix: do not encodeURIComponent booleans or null objects

### PR DESCRIPTION
**Pull request by adamkdean**
This PR ensures that booleans do not pass through `encodeURIComponent` in the `_encodeObjectKeys` function.

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -192,6 +192,8 @@ module.exports = function (APIWrapper) {
           array.push(arrayItem)
         }
         result[key] = array
+      } else if (typeof result[key] === 'boolean') {
+        // do nothing
```
Comment (jimlambie) [inline lib/helpers.js]:
```
      } else if (typeof object[key] === 'boolean') {
        result[key] = object[key]

```

No?

Comment (adamkdean) [reply lib/helpers.js]: We could do that, yes. I'll fix it.

Comment (jimlambie) [reply lib/helpers.js]: I think if you do nothing ,you get `filter={"tmdbId":{}}`

**Comment by adamkdean** [comment]
@jimlambie fixed that assignment. Mistakenly thought result object was being mutated.

**Pull request merged**